core/templates/dev/head/pages/profile/Profile.js

```diff
@@ -88,7 +88,7 @@ oppia.controller('Profile', [
 
       $scope.changeSubscriptionStatus = function() {
         if ($scope.userNotLoggedIn) {
-          UserService.getLoginAndLogoutUrls().then(
+          UserService.getLoginUrl().then(
             function(urlObject) {
```

Comment:
Does this still need to be an object? The name of get_login_url() suggests that a string is returned (though maybe we should rename it to getLoginUrlAsync()).

Reply:
Done.